runtime/onert/api/nnapi/wrapper/ANeuralNetworksExecution.cc

```diff
@@ -136,7 +136,7 @@ bool ANeuralNetworksExecution::setInput(uint32_t index, const ANeuralNetworksOpe
   try
   {
     onert::ir::IOIndex input_index{index};
-    const auto shape =
+    const auto &shape =
       (type != nullptr) ? NNAPIConvert::getShape(type) : _execution->getInputShape(input_index);
```

Comment:
IMHO, this approach may not solve copy well. I think we have to resolve the two problems.
- `NNAPIConvert::getShape()` can be optimized by RVO. Adding reference actually interferes with RVO. So, we don't have to add any reference where RVO can be applied.
- `getInputShape()`'s return type is value type and the returned object's type is a lvalue reference. So, `getInputShape()` always calls it own copy constructor when it is called.

Reply:
Maybe you're right. This PR is wrote to remove analyzer's warning.

Reply:
I see. Does the analyzer remove the warning just by adding reference?

Reply:
> I see. I approved this PR because the existing code already has similar problems. Does the analyzer remove the warning just by adding reference?

I tested analyzer and confirmed it removes the warning.